setup.cfg

```diff
@@ -4,3 +4,8 @@ detailed-errors=True
 with-coverage=True
 cover-package=flask_mongorest
 cover-erase=True
+
+[flake8]
+ignore=W391,W602,E501,E121,E122,E123,E124,E126,E127,E128,E131,E2,E301,E302,E303,E402,E5,E702,E731,F401,F403,F812,F841,F999
+exclude=build,dist,docs,node_modules,venv,lib,scripts,.tox
+max-complexity=185
```

Comment:
why? you copied these from closeio/ but those values were chosen only b/c we had so much existing issues in that repo. when adding flake8 to a new repo we should add only the fewest ignore's needed and the lowest max-complexity that we can
also some of your `exclude` paths don't make any sense in this repo